doc/developer/fast-compiles.md

```diff
@@ -0,0 +1,91 @@
+# Fast compiles
+
+Rust has notoriously slow compile times for the simplest of projects, and
+Materialize is not a simple project. This document attempts to capture the
+various lessons learned by Materialize engineers when optimizing machines for
+compiling Rust.
+
+## Run Linux
+
+The most important step you can take to speed up Rust compilation times is to
+run Linux rather than macOS.
+
+First, your money goes much farther if you don't pay the Apple tax. For a given
```

Comment:
further

Reply:
https://english.stackexchange.com/a/152207

Reply:
> The usage that is most likely to sound wrong to 21st-century speakers is "farther" in the figurative sense.

From your link, and I concur!

Reply:
(I don't really care either way though; use what you like)

Reply:
Heh, you're welcome to change it to further if you like, because we both agree that's valid. But "money goes farther" doesn't sound wrong to my ear so don't expect me to notice in the future. I feel the same way about less vs. fewer, split infinitives... most of these rules are made up by English teachers!